sql/core/src/main/scala/org/apache/spark/sql/Column.scala

```diff
@@ -1008,7 +1009,7 @@ class Column(val expr: Expression) extends Logging {
   def cast(to: String): Column = cast(CatalystSqlParser.parseDataType(to))
 
   /**
-   * Returns an ordering used in sorting.
+   * Returns a sort expression based on the descending order of the column.
```

Comment:
Do you want to include example usages, as in the python documentation? E.g. for rlike,
```
   * {{{
   *   // find names that start with "Al"
   *   scala> df.filter( $"name".like("Al%") ).collect()
   *   Array([Alice,1])
   * }}}
```

It would also be helpful for the `startsWith` and `endsWith` functions, where there are two versions, e.g. `startsWith(other: Column)` and `startsWith(literal: String)`.